bin/plugin/commands/changelog.js

```diff
@@ -188,6 +265,62 @@ function getIssueType( issue ) {
 	return candidates.length ? candidates.sort( sortType )[ 0 ] : 'Various';
 }
 
+/**
+ * Removes any `[Feature]` prefix from a given label.
+ *
+ * @param {string} label The label potentially containing a prefix.
+ *
+ * @return {string} the label without the prefix.
+ */
+function stripFeaturePrefix( label ) {
+	return label.replace( '[Feature] ', '' );
+}
+
+/**
+ * Returns the most appropriate feature category for the given issue based
+ * on a basic heuristic.
+ *
+ * @param {IssuesListForRepoResponseItem} issue Issue object.
+ *
+ * @return {string} the feature name.
+ */
+function getIssueFeature( issue ) {
+	const labels = issue.labels.map( ( { name } ) => name );
+
+	const featureCandidates = mapLabelsToFeatures( labels );
+
+	// 1. Prefer explicit mapping of label to feature.
+	if ( featureCandidates.length ) {
+		// Get occurances of the feature labels.
+		const featureCounts = countBy( featureCandidates );
+
+		// Check which matching label occurs most often.
+		const rankedFeatures = Object.keys( featureCounts ).sort(
+			( a, b ) => featureCounts[ b ] - featureCounts[ a ]
+		);
+
+		// Return the one that appeared most often.
+		return rankedFeatures[ 0 ];
+	}
+
+	// 2. `[Feature]` labels
+	const featureSpecificLabel = getFeatureSpecificLabels( labels );
+
+	if ( featureSpecificLabel ) {
+		return stripFeaturePrefix( featureSpecificLabel );
+	}
+
+	// 3. Block specific labels.
+	const blockSpecificLabels = getIsBlockSpecificIssue( labels );
+
+	if ( blockSpecificLabels ) {
+		return 'Block Library';
```

Comment:
A nice follow-up would be to add a subgroup for the block library and detect the block name if possible. Sometimes there are multiple entries for the same block type.

Reply:
Followup created.